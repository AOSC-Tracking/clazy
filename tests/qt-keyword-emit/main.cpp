#include <QtCore/QObject>
#include <QtCore/QString>

class MyObj : public QObject
{
public:
Q_SIGNALS:
    void signal1();

    void test()
    {
        emit signal1();
    }
};
