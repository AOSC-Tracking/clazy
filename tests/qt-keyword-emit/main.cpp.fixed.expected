#include <QtCore/QObject>
#include <QtCore/QString>

class MyObj : public QObject
{
public:
Q_SIGNALS:
    void signal1();

    void test()
    {
        Q_EMIT signal1();
    }
};
